SYCL/Plugin/interop-level-zero.cpp

```diff
@@ -7,22 +7,25 @@
 #include <CL/sycl.hpp>
 // clang-format off
 #include <level_zero/ze_api.h>
-#include <CL/sycl/backend/level_zero.hpp>
+#include <sycl/ext/oneapi/backend/level_zero.hpp>
```

Comment:
We state Level Zero like an extension.
Shouldn't we add corresponding macro SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO in sycl/include/CL/sycl/feature_test.hpp like all other extensions and enable the test only when extension is available?

Reply:
The macro is there already https://github.com/intel/llvm/blob/e66216673f599d85c056bc3a9dd7ddfd5544ff3d/sycl/include/CL/sycl/feature_test.hpp#L30

Reply:
I don't think we have to test it in this test, since we know or implementation supports it.

Reply:
But not all implementations and build versions may support it (e.g. HIP or CUDA builds may miss this extension one day).
I think it is good practice to wrap extension test with the check.

Reply:
Anyways, I have add the check for SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO